### DISCUSSION TITLE
arcstat.py with color?

### DISCUSSION THREAD

**Discussion by richardm1** [openzfs/zfs#14947] [Ideas]
Asking as someone who doesn't know Python, how hard would it be to implement colors within the arcstat script? I'm not suggesting anything fancy like colors that change based on values; just having some simple static column coloration to make them easier to read.

I've looked for modified arcstat.py scripts and haven't found anything.  I figure if this were low hanging fruit someone would've done it already. 🙂

**Comment by rincebrain** [comment]
Nope, it's low-hanging fruit, it's just a minority of people have enough of a desire for colored text output to actually implement it.
